src/components/PopupWithConfirm.js

```diff
@@ -0,0 +1,30 @@
+import Popup from "./Popup";
+
+export default class PopupWithConfirm extends Popup {
+  constructor({ modalSelector }) {
+    super({ modalSelector });
+    this._modalForm = this._modalElement.querySelector(".modal__form");
+    this._submitButton = this._modalForm.querySelector(".modal__button-save");
+    this._submitButtonTextContent = this._submitButton.textContent;
+  }
+
+  setSubmitFunc(SubmitFunc) {
+    this._SubmitFunc = SubmitFunc;
+  }
```

Comment:
`SubmitFunc`

Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)

Reply:
My remark hasn't been corrected https://github.com/bencorpus/se_project_aroundtheus/pull/1#discussion_r2029914907